.github/renovate.json5

```diff
@@ -0,0 +1,19 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  description: "Renovate config for LavaMoat monorepo",
+  extends: [
+    "config:js-app", // everything gets pinned except peer deps (plus a bunch of other reasonable defaults)
+    "group:definitelyTyped", // groups all `@types/*` packages together
+    "helpers:pinGitHubActionDigests", // pins SHAs of GitHub actions
+    "workarounds:typesNodeVersioning", // workaround for `@types/node` versioning
+    ":automergePatch", // automatically merges "patch" and "pin" PRs on check success
+    ":automergeMinor", // automatically merges "minor" PRs on check success
+    ":automergeDigest", // automatically merges "digest" PRs on check success
+    ":enableVulnerabilityAlerts", // enables GitHub vulnerability alerts
+    ":rebaseStalePrs", // automatically rebase stale PRs against main
+    ":semanticCommits", // use semantic commits
+    ":semanticPrefixFixDepsChoreOthers", // use prefix "fix" for prod deps and "chore" for everything else
+    "group:linters", // group lint-related packages together
+  ],
+  transitiveRemediation: true,
```

Comment:
does this config opt mean renovate[bot] will take over Dependabot transitive PRs? like

e.g. latest one https://github.com/LavaMoat/LavaMoat/commit/e821e3e07e087c27b1dfed63550c9ebcf7080188 (where `mkdirp` being picked up in few places)

```console
lavamoat-monorepo@0.0.0 /Users/leo/Documents/GitHub/LavaMoat
├─┬ @lavamoat/yarn-plugin-allow-scripts@1.0.0 -> ./packages/yarn-plugin-allow-scripts
│ └─┬ @yarnpkg/core@3.5.3
│   ├── mkdirp@0.5.6
│   └─┬ tar@6.1.15
│     └── mkdirp@1.0.4
├─┬ lavamoat-browserify@15.7.1 -> ./packages/browserify
│ └─┬ source-map-explorer@2.5.3
│   └─┬ temp@0.9.4
│     └── mkdirp@0.5.6 deduped
└─┬ lerna@7.1.5
  ├─┬ @lerna/create@7.1.5
  │ └─┬ tar@6.1.11
  │   └── mkdirp@1.0.4
  └─┬ tar@6.1.11
    └── mkdirp@1.0.4
```

Reply:
That's the idea, yeah.  Only if there is a vuln.  I think GitHub vuln alerts have to be enabled on the repo, too.